INDEX

```diff
@@ -325,5 +325,13 @@
         "description": "DVR server for the Channels apps. Makes digital recordings of broadcast television using HDHomeRun network tuners. (Requires Subscription)",
         "primary_pkg": "",
         "official": false
+    },
```

Comment:
This also seems out of place, it needs to be indented with 4 spaces. And the same is true for the lines below